site/src/comparison.rs

```diff
@@ -299,6 +299,11 @@ impl ComparisonSummary {
         self.average(self.improvements())
     }
 
+    /// The average change as a percent
```

Comment:
While you're in here, changing all uses of `average` in this code to `arith_mean` or similar wouldn't be a bad thing... though I understand if you don't want to do it in this PR.

Reply:
Good idea, I'll change it.